src/chains/ethereum/options/src/index.ts

```diff
@@ -18,14 +17,22 @@ import {
 } from "@ganache/options";
 import { UnionToIntersection } from "./helper-types";
 
-export type EthereumOptions = {
+type EthereumConfig = {
   chain: ChainConfig;
   database: DatabaseConfig;
   logging: LoggingConfig;
   miner: MinerConfig;
   wallet: WalletConfig;
 };
 
+export const EthereumDefaults: Defaults<EthereumConfig> = {
```

Comment:
This was a personal choice that it felt that `ethereumDefaults` was out of place and inconsistent with the several other exports here. Further, there aren't many examples of the codebase of an exported const object's naming. outside of these default objects, you have a `params` in `@ganache/ethereum-utils` and 4 `WeakMap` variables in `json-rpc-base-types.ts`. So there wasn't really a precedent set for the styling here other than these default objects (seems that I forgot one in `@ganache/core/src/options`).

I can change them all back to be camelcase, but camelcase const objects feel mutable and modular rather than global/immutable

--

While I still standby the above, I remembered that the driving change of this happened when changing `@ganache/flavors/index.ts` which has `export const DefaultOptionsByName`. So since I prefer the title casing here, I switched those to adhere to that rather than switching it to camelcasing

Reply:
Probably just a case of me not being consistent. I don't really think of this particular value as a "constant", but I guess we could think of it that way. For simple constants I've been using UPPERCASE (See constants.ts in the utils package).

You can leave this one as is and Resolve this conversastion.